src/app/home/feature-box/feature-box.component.scss

```diff
@@ -0,0 +1,66 @@
+@import "../../_variables.scss";
+
+// inline image feature box
+.feature-box {
+  background: $gray-dark1;
+  text-align: center;
+  margin: 20px 16px 0;
+  padding: 18px 13px;
+  border-radius: 3px;
+
+  h2 {
+    font-weight: bold;
+    font-size: 36px;
+    line-height: 42px;
+  }
+
+  p {
+    font-size: 18px;
+    line-height: 140%;
+  }
+
+  img {
+    border-radius: 3px;
+    width: 95%;
+    max-width: 450px;
+    max-height: 500px;
+  }
+}
+
+@media only screen and (min-width: 768px) { // desktop
+  .feature-box {
+    margin: 20px 60px 0;
+    padding: 35px 56px;
+  }
+}
+
+// large feature box
+.large-feature-box {
+  margin: 20px 130px 0;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 30px;
+  text-align: left;
+
+  div {
```

Comment:
I would suggest adding a class to this element and using that for this selector instead of `div`. Generally, using all classes and no tags for styling makes your CSS and HTML the most flexible, but sometimes styling on semantic tags like `p` or `img` can be helpful (though I still try to avoid styling on _any_ tags for the most part). I'd at least always stay away from using `span` or `div` in CSS though, since they carry no inherent semantic meaning so using them in CSS tightly constrains future changes to the markup without needing to also refactor the CSS (`div` and `span` are like wildcard elements, their semantic meaning comes from the class you put on them). For instance, if you need to add a wrapper `div` inside of `large-feature-box` in the future, it's going to get top padding even if it's not the `div` this style was originally intended for (also your `h2` and `p` styles will only apply if there is a `div` between them and the `large-feature-box` so their wrapping `div` can never be removed).